deployments/liqo/templates/liqo-fabric-daemonset.yaml

```diff
@@ -43,6 +43,7 @@ spec:
           command: ["/usr/bin/liqo-fabric"]
           args:
           - --nodename=$(NODE_NAME)
+          - --enable-arp=true
```

Comment:
Should we make it configurable? The same for the other images

Reply:
I don't think so. It is used just for debugging at the moment and maybe could be useful in Google Cloud, but the controller to populate the FIB has not been implemented yet (and we still don't know if we need it).